datadog_checks_dev/datadog_checks/dev/tooling/commands/env/start.py

```diff
@@ -66,7 +66,8 @@ def start(ctx, check, env, agent, python, dev, base, env_vars):
 
     if env not in envs:
         echo_failure('`{}` is not an available environment.'.format(env))
-        echo_info('See what is available via `ddev env ls {}`.'.format(check))
+        echo_info('Available environments for {}:\n\t{}'.format(check, "\n\t".join(envs)))
```

Comment:
```suggestion
        echo_info('Available environments for {}:\n    {}'.format(check, '\n    '.join(envs)))
```

Reply:
\t does not work on windows?

Reply:
Spacing is not consistent across terminals